src/Filesystem/File.php

```diff
@@ -556,11 +556,12 @@ public function mime()
         }
         if (function_exists('finfo_open')) {
             $finfo = finfo_open(FILEINFO_MIME);
-            $finfo = finfo_file($finfo, $this->pwd());
             if (!$finfo) {
                 return false;
             }
-            list($type) = explode(';', $finfo);
+            $mime = finfo_file($finfo, $this->pwd());
```

Comment:
yeah sorry i got mislead by old c habbits of docs reading where finfo_file says it returns string

so full should be like
```PHP
$finfo = finfo_open(FILEINFO_MIME);
if (!$finfo) {
     return false;
}
$mime = finfo_file($finfo, $this->pwd());
if (!$mime) {
    finfo_close($finfo);
    return false;
}
finfo_close($finfo);
list($type) = explode(';', $mime);
return $type;
```
or anything else to change you see with your eyes?